templates/minimal/README.md

```diff
@@ -1,13 +1,93 @@
-# Minimal Template
+<div align="center">
 
-This is a minimal template for creating a blockchain using the Polkadot SDK.
+# Polkadot SDK's Minimal Template
 
-# Docs
+<img height="70px" alt="Polkadot SDK Logo" src="https://github.com/paritytech/polkadot-sdk/raw/master/docs/images/Polkadot_Logo_Horizontal_Pink_White.png#gh-dark-mode-only"/>
+<img height="70px" alt="Polkadot SDK Logo" src="https://github.com/paritytech/polkadot-sdk/raw/master/docs/images/Polkadot_Logo_Horizontal_Pink_Black.png#gh-light-mode-only"/>
 
-You can generate and view the [Rust
-Docs](https://doc.rust-lang.org/cargo/commands/cargo-doc.html) for this template
-with this command:
+> This is a minimal template for creating a [Substrate](https://substrate.io/) blockchain.
+> 
+> This template is automatically updated after releases in the main [Polkadot SDK monorepo](https://github.com/paritytech/polkadot-sdk).
+
+</div>
+
+## Getting Started
+
+🛠️ Depending on your operating system and Rust version, there might be additional
+packages required to compile this template.
+
+👉 Check the
+[Substrate Install](https://docs.substrate.io/install/) instructions for your platform for
+the most common dependencies.
+
+### Build
+
+🔨 Use the following command to build the node without launching it:
+
+```sh
+cargo build --release
+```
+
+🐳 Alternatively, build the docker image:
 
 ```sh
-cargo doc -p minimal-template --open
+docker build . -t polkadot-sdk-minimal-template
 ```
+
+### Single-Node Development Chain
+
+👤 The following command starts a single-node development chain:
+
+```sh
+./target/release/minimal-template-node --dev
+
+# docker version:
+docker run --rm polkadot-sdk-minimal-template --dev
+```
+
+Development chains:
+
+- 🧹 Do not persist the state.
+- 💰 Are preconfigured with a genesis state that includes several prefunded development accounts.
+- 🧑‍⚖️ Development accounts are used as default validator authorities and a `sudo` account.
```

Comment:
Does the template actually have this? needs to be double checked

Reply:
The accounts are prefunded, and I sent a `sudo` extrinsic from Alice and it worked.
Not sure what to check in terms of `default validator authorities`.

![Screenshot 2024-05-31 at 10 11 31](https://github.com/paritytech/polkadot-sdk/assets/12039224/e01fbe16-f221-44ad-82ad-e7d704ef9063)

Reply:
there are no validators in this setup, you can remove it.

Reply:
Removed.